spec/Powertrain/PowerSource/ElectricMotor.vspec

```diff
@@ -10,3 +11,99 @@
 #
 
 
+- MaxPower:
+  datatype:  uint16
+  type: attribute
+  default: 0
+  unit: kW
+  description: Peak power, in kilowatts, that motor(s) can generate.
+
+- MaxTorque:
+  datatype:  uint16
+  type: attribute
+  default: 0
+  unit: N.m
+  description: Peak power, in newton meter, that the motor(s) can generate.
+
+- MaxRegenPower:
+  datatype:  uint16
+  type: attribute
+  default: 0
+  unit: kW
+  description: Peak regen/brake power, in kilowatts, that motor(s) can generate.
+
+- MaxRegenTorque:
+  datatype:  uint16
+  type: attribute
+  default: 0
+  unit: N.m
+  description: Peak regen/brake torque, in newton meter, that the motor(s) can generate.
+
+
+#
+# Motor
+#
+- Motor:
+  type: branch
+  description: motor signals
+
+
+#
+# Motor rotations per minute
+#
+- Motor.Speed:
```

Comment:
RotationalSpeed?

Reply:
Agree, it should be something different than Speed.

Reply:
Maybe direclty "rpm" ?

Reply:
Motor.RPM would also be acceptable for me.  (I think "RPM" it is very well known concept)

Reply:
I also like RPM. Maybe later we should give some love to spec/Powertrain/PowerSource/CombustionEngine.vspec  to make it consistent